google/cloud/spanner_dbapi/connection.py

```diff
@@ -126,6 +134,15 @@ def autocommit(self):
         """
         return self._autocommit
 
+    @property
+    def buffer_ddl_statements(self):
```

Comment:
Is this a read-only property? As in; can it only be set when a connection is created, and never changed after that? If so, is that something that we can/should document? Or is it clear from the context?

Reply:
Yes its a read only property and there are no setters for it but in python the private variable can also be overridden.